tests/test_omop_converter.py

```diff
@@ -0,0 +1,216 @@
+from __future__ import annotations
+
+import piton.extractors.omop_converter
+import piton
+import datetime
+
+
+def test_pre_birth() -> None:
+    patient = piton.Patient(
+        patient_id=123,
+        events=[
+            piton.Event(start=datetime.datetime(1999, 7, 2), code=1234),
+            piton.Event(start=datetime.datetime(1999, 7, 9), code=4216316),
+            piton.Event(start=datetime.datetime(1999, 7, 11), code=12345),
+        ],
+    )
+
+    expected = piton.Patient(
+        patient_id=123,
+        events=[
+            piton.Event(start=datetime.datetime(1999, 7, 9), code=4216316),
+            piton.Event(start=datetime.datetime(1999, 7, 11), code=12345),
+        ],
+    )
+
+    assert (
+        piton.extractors.omop_converter._remove_pre_birth(patient) == expected
+    )
+
+
+def test_remove_small() -> None:
+    patient = piton.Patient(
+        patient_id=123,
+        events=[
+            piton.Event(start=datetime.datetime(1999, 7, 2), code=1234),
+            piton.Event(start=datetime.datetime(1999, 7, 9), code=4216316),
+            piton.Event(start=datetime.datetime(1999, 7, 11), code=12345),
+            piton.Event(start=datetime.datetime(1999, 7, 13), code=12345),
+        ],
+    )
+
+    invalid = piton.Patient(
+        patient_id=123,
+        events=[
+            piton.Event(start=datetime.datetime(1999, 7, 9), code=4216316),
+            piton.Event(start=datetime.datetime(1999, 7, 11), code=12345),
+        ],
+    )
+
+    assert (
+        piton.extractors.omop_converter._remove_small_patients(patient)
+        == patient
+    )
+
+    assert (
+        piton.extractors.omop_converter._remove_small_patients(invalid) is None
+    )
+
+
+def test_move_to_day_end() -> None:
+    patient = piton.Patient(
+        patient_id=123,
+        events=[
+            piton.Event(start=datetime.datetime(1999, 7, 2), code=1234),
```

Comment:
```suggestion
            piton.Event(start=datetime.datetime(1999, 7, 2), code=4321),
```

Reply:
Differentiating the two codes makes it clearer which is which following the sort